.NET/Microsoft.Recognizers.Text.Sequence/SequenceRecognizer.cs

```diff
@@ -89,7 +91,7 @@ public IModel GetEmailModel(string culture = null, bool fallbackToDefaultCulture
 
         public IModel GetURLModel(string culture = null, bool fallbackToDefaultCulture = true)
         {
-            return GetModel<URLModel>(Culture.English, fallbackToDefaultCulture);
+            return GetModel<URLModel>(culture, fallbackToDefaultCulture);

```

Comment:
This is a regression compared to the previous PR. It will fail for other languages that are not EN/CN/JP.
Can you add back that if/else that returns the customized extractor for CN/JP and the base English one for others?